.env.local.default

```diff
@@ -32,6 +32,7 @@ VITE_APP_HOST=localhost
 VITE_APP_PORT=3000
 VITE_ZENDESK_ENABLED=false
 VITE_ZENDESK_KEY=
+VITE_ZENDESK_AUTHENTICATION_ENABLED=false
```

Comment:
When will a user need to use `VITE_ZENDESK_AUTHENTICATION_ENABLED`? 

What if, `VITE_ZENDESK_ENABLED` is `true` and `VITE_ZENDESK_AUTHENTICATION_ENABLED` is `false`. How will it work?

Reply:
So authentication is not required to use Zendesk, if it's not enabled users can still contact support but they have to provide their name and email, with Zendesk authentication enabled end users  can skip that step since the information is already stored in the token used to authenticate them. 

An added benefit of Zendesk is that they can view their conversation on multiple devices or even after cleaning the cache in their devices.